engine/src/flutter/shell/platform/embedder/embedder_semantics_update.cc

```diff
@@ -126,6 +126,44 @@ FlutterSemanticsFlag SemanticsFlagsToInt(const SemanticsFlags& flags) {
   return static_cast<FlutterSemanticsFlag>(result);
 }
 
+const FlutterSemanticsFlags convertToFlutterSemanticsFlags(
```

Comment:
In C++ style, functions and methods are `UpperCamelCase`, except (optionally) for simple getters, which are `lower_snake_case`.